lib/xlsx/xform/table/table-xform.js

```diff
@@ -64,6 +64,7 @@ class TableXform extends BaseXform {
         this.model = {
           name: attributes.name,
           displayName: attributes.displayName || attributes.name,
+          ref: attributes.ref,
```

Comment:
same as tableRef, why was it added?

Reply:
It was required during the creation of the table or otherwise the validator complained and crashed the execution 🤷‍♂️